chia/util/puzzle_compression.py

```diff
@@ -180,6 +183,7 @@ def solve(args: List[Union[PuzzleRepresentation, Program]], solution_dict: Dict[
 
 
 class KnownPuzzles:
+    version: uint16 = 1  # This needs to be incremented whenever the map below changes
```

Comment:
Don't use a global. Create (in another file) an instance of `KnownPuzzles`, populate it with the v1 puzzles, and `import` that instance when needed.

Never, ever use a global unless you have an unbelievably good reason. This ain't it.

Reply:
tl;dr, :no_entry_sign: :globe_with_meridians: